Week2/6.2.2000_run.py

```diff
@@ -0,0 +1,20 @@
+import time
+
+def timer(f, *args, **kwargs):
+    """
+    timer decorator, which prints the time taken to run a function.
+    :param f: function to be timed.
+    :param args: arguments to be passed to f
+    :param kwargs: keyword arguments to be passed to f
+    :return: wrapper function
+    """
+    start_time = time.time()
```

Comment:
time() method is not accurate enough.
Read the time module documentation in order to use a better function.